### PR TITLE
release: 2.62

### PR DESCRIPTION
**Pull request by ernestl**
Merge changelogs back into master, please use a regular “merge” to merge it.

### CODE REVIEW

**Review by zyga — APPROVED** [review]
This looks good

File: NEWS.md [hunk 1]
```diff
@@ -1,15 +1,80 @@
...
-* Started support for snap services to show real status of user daemons
-
-# Next:
-* state: add support for notices (from pebble)
-* daemon: add notices to the snapd API under `/v2/notices` and `/v2/notice`
-* Mandatory device cgroup for all snaps using bare and core24 base as well as future bases
-* Added API route for creating recovery systems: POST to `/v2/systems` with action `create`
-* Added API route for removing recovery systems: POST to `/v2/systems/{label}` with action `remove`
-* Support for user daemons by introducing new control switches --user/--system/--users for service start/stop/restart
-* client,daemon: expose features supported/enabled in `/v2/system-info`
```
Comment (bboozzoo) [inline NEWS.md:12]: Should In progress items be brought back?

**Comment by bboozzoo** [comment]
On second though, I think this doesn't solve the problem of tags appearing on master.

File: NEWS.md (hunk 1 above)
Comment (Meulengracht) [reply NEWS.md:12]: Yes, they shouldn't be deleted

**Review by Meulengracht — APPROVED** [review]

**Comment by zyga** [comment]
@ernestl let's merge the entire release branch so that we fix the situation around tags.

**Comment by bboozzoo** [comment]
I've reopened my branch at https://github.com/snapcore/snapd/pull/13749

**Pull request closed without merging**